packages/components/src/molecules/Alert/Alert.tsx

```diff
@@ -0,0 +1,90 @@
+import type { HTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+import { useCallback } from 'react'
+
+import { Button, Link, Icon } from '../../index'
+
+import { X } from '../../assets'
+
+import type { ReactNode, MouseEvent } from 'react'
+
+export interface AlertProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'role'> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Icon component for additional customization.
+   */
+  icon?: ReactNode
+  /**
+   * Enables dismissible feature.
+   */
+  dismissible?: boolean
+  /**
+   * The href and label used at the link.
+   */
+  link?: {
+    to: string
+    text: string
+  }
```

Comment:
I think it is essential to add the target attribute here because the user wants to open the link in a new tab.